### ISSUE TITLE
Level: "The Matrix"

### ISSUE THREAD

**Issue by colonelfazackerley** [andybalaam/rabbit-escape#63]
Curious to know how hard you guys find this. I can't judge, as I have been tweaking too long.
```
:name=The Matrix
:description=There is no spoon.
:num_rabbits=24
:num_to_save=24
:rabbit_delay=16,8,4,2,1
:bridge=3
:bash=5
:climb=1
:music=tryad-witness
     Q         
###         ###
##  #  #  #  ##
# )( )( )( )( #
  () () () ()  
 #  #  #  #  # 
  )( )( )( )(  
  () () () ()  
 #  #  #  #  # 
  )( )( )( )(  
# () ()O() () #
##  #  #  #  ##
###         ###
```
(Needs dev build: last release does not have the variable delay)

**Comment by colonelfazackerley** [comment]
@tttppp, you can use the jar here 
https://github.com/andybalaam/rabbit-escape/releases/tag/v0.5
to try this level.

**Comment by tttppp** [comment]
Took me about 5 minutes, but plenty of restarts!  The timing and placement needs to be spot on to solve :-)

**Comment by colonelfazackerley** [comment]
merged to master with f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a
